app/controllers/v1/resources_controller.rb

```diff
@@ -12,7 +12,7 @@ class ResourcesController < ApplicationController
     def create
       begin
         UpdateStacksFilesService.new(@cocina_object).write!
-        UpdatePurlMetadataService.new(@cocina_object).write!
+        UpdatePurlMetadataService.new(@cocina_object, @purl).write!
```

Comment:
I think this should be `only: [:cocina, :xml]`. We don't want this action to touch `meta.json`

Reply:
DOR services app automatically sends the Released#create API call right after the Purl#updates call, so this should be taken care of by that: https://github.com/sul-dlss/dor-services-app/blob/main/app/services/publish/metadata_transfer_service.rb#L77-L85

Reply:
And we don't want to change dor-services to send that information with the cocina data?

Also, is there any harm in writing the same file twice? I guess to me, it seems like there's some benefits to being proactive on the purl-fetcher side in order to avoid half-baked objects.

Reply:
>

> And we don't want to change dor-services to send that information with the cocina data?

Versioned vs unversioned data.  We are splitting purl-fetcher into two APIs, one that says "put these files and data in place" (previously publish/shelve) and one that says "send this stuff on to indexers"

> Also, is there any harm in writing the same file twice? I guess to me, it seems like there's some benefits to being proactive on the purl-fetcher side in order to avoid half-baked objects.

Yes, I think it's confusing if one of the endpoints does publish/shelve/release and one just does release. We're working to simplify the process into two distinct services.

Reply:
🤷‍♂️ ok, as long as it makes sense to you.